src/components/Container/README.md

```diff
@@ -1,5 +1,169 @@
 # Container
 
+Container contains content.
+
+```vue
+<template>
+	<m-container>
+		container content
+	</m-container>
+</template>
+
+<script>
+import { MContainer } from '@square/maker/components/Container';
+
+export default {
+	components: {
+		MContainer,
+	},
+};
+</script>
+```
+
+## Labels
+
+Labels can be set via props.
+
+```vue
+<template>
+	<m-container
+		label="container label"
+		sublabel="container sublabel"
+		requirement-label="container requirement label"
+	>
+		container content
+	</m-container>
+</template>
+
+<script>
+import { MContainer } from '@square/maker/components/Container';
+
+export default {
+	components: {
+		MContainer,
+	},
+};
+</script>
+```
+
+Labels can also be set via slots.
+
+```vue
+<template>
+	<m-container
+		label="container label"
```

Comment:
Want to remove the label props from this example?

Reply:
yup, good catch